main/main.go

```diff
@@ -0,0 +1,193 @@
+package main
+
+import (
+	"fmt"
+	"github.com/featureform/api"
+	"github.com/featureform/coordinator"
+	"github.com/featureform/ffsync"
+	help "github.com/featureform/helpers"
+	"github.com/featureform/logging"
+	"github.com/featureform/metadata"
+	dm "github.com/featureform/metadata/dashboard"
+	"github.com/featureform/metadata/search"
+	"github.com/featureform/metrics"
+	pb "github.com/featureform/proto"
+	"github.com/featureform/runner"
+	"github.com/featureform/scheduling"
+	"github.com/featureform/serving"
+	ss "github.com/featureform/storage"
+	"github.com/joho/godotenv"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	"net"
+	"net/http"
+	"os"
+	"time"
+)
+
+func main() {
+	locker := ffsync.NewMemoryLocker()
+	mstorage := ss.NewMemoryStorageImplementation()
+	storage := ss.MetadataStorage{
+		Locker:  &locker,
+		Storage: &mstorage,
+	}
+	meta := scheduling.NewTaskMetadataManager(storage, ffsync.NewMemoryOrderedIdGenerator())
+
+	local := help.GetEnvBool("FEATUREFORM_LOCAL", true)
+	/****************************************** API Server ************************************************************/
+	err := godotenv.Load(".env")
+	apiPort := help.GetEnv("API_PORT", "7878")
+	metadataHost := help.GetEnv("METADATA_HOST", "localhost")
+	metadataPort := help.GetEnv("METADATA_PORT", "8080")
```

Comment:
should be able to reuse the vars for `METADATA_PORT`, `SERVING_HOST`, `SERVING_PORT` instead of refetching them in the lines below. (59, 138, 139)